runtime/Cpp/runtime/src/atn/SemanticContext.h

```diff
@@ -144,6 +144,8 @@ namespace atn {
    */
   class ANTLR4CPP_PUBLIC SemanticContext::Operator : public SemanticContext {
   public:
+    ~Operator();
```

Comment:
It is virtual because SemanticContext has a virtual destructor. In our codebase I would have written it is as "~Operator override;". Our reasoning on virtual functions is "virtual" keyword is superfluous when using "override" or "final", and "override" is superfluous when using "final", so our rule is to pick one and one only on a virtual function.

I was trying (not that it's obvious!) to stay consistent with the files I was touching, so happy to say "virtual ~Operator()" or "~Operator() override" as you prefer.